### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by myuriy81**
[DEMO LINK](https://myuriy81.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://myuriy81.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,79 @@
...
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 50px;
+  display: flex;
+}
+
+.logo {
+  align-self: center;
+  justify-content: center;
```
Comment (TarasHoliuk) [inline src/style.css:27]: it's redundant here:

image

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+  text-transform: uppercase;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  color: #000;
+  height: 60px;
+}
+
+.nav_link:hover {
+  color: #00acdc;
```
Comment (TarasHoliuk) [inline src/style.css:57]: move colors that you use a few times into variables and use them

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Well done 👍 
Only 2 small comments 🙂

File: src/style.css (hunk 1 above)
Comment (myuriy81) [reply src/style.css]: +

Comment (myuriy81) [reply src/style.css]: Готово

File: src/style.css (hunk 2 above)
Comment (myuriy81) [reply src/style.css]: Готово. Шо робити далі?

**Review by nazarmatsevych — APPROVED** [review]
Good work!

**Pull request closed without merging**